lib/EditorJs.tsx

```diff
@@ -123,10 +124,22 @@ class EditorJsContainer extends React.PureComponent<Props> {
       })
   }
 
+  constructor(props: EditorJsProps) {
+    super(props)
+
+    const randomString = () => {
+      const min = 1000 * 1000 * 1000
+      const max = 9999 * 1000 * 1000
+      return (Math.round(Math.random() * (max - min * min)) + min).toString(36)
+    }
```

Comment:
```suggestion
    this.holderID = `editor-js-${Date.now().toString(36)}`
```

If you don't mind, how about generating string based on Date object?

Reply:
This makes sense, but in theory, couldn't this lead to a collision if two instances are added at the same *exact* time? What about adding random jitter of about 1000 to the Date.now() value?

`Date.now() + Math.floor(Math.random() * 1000)`

Reply:
Wow! that's great!